libs/wingsdk/src/cloud/queue.ts

```diff
@@ -19,6 +19,12 @@ export interface QueueProps {
    */
   readonly timeout?: Duration;
 
+  /**
+   * How long a queue retains a message.
+   * @default Duration.fromHours(1)
```

Comment:
@fynnfluegge why did you choose 1 hour and not 30 seconds?

Reply:
AWS SQS for example has minimum 1 minute retention period and maximum of 14 days. To me 1 hour is always a good default for retention. We have timeout 30s default, if we choose for example 1 min retention then the message is lost if the function throws an error once. Maybe at least in the simulator we can choose several minutes for default? Running the simulator for hours is not a realistic scenario 😁

Reply:
Default value for AWS SQS is 4 days retention 😁

Reply:
This default is sounds pretty sensible to me but its only used for the sim so I dont think we want to annotate this on the `cloud.queue`

Reply:
I agree, but we should also remove the annotation from timeout, right? 😁